charts/kyverno/templates/_helpers/_deployment.tpl

```diff
@@ -1,7 +1,7 @@
 {{/* vim: set filetype=mustache: */}}
 
 {{- define "kyverno.deployment.replicas" -}}
-  {{- if eq (int (default 1 .)) 0 -}}
+  {{- if and (kindIs "int" .)  (eq (int (default 1 .)) 0) -}}
```

Comment:
Only difference i can think of is when the input value is `null` 🤔

Reply:
I am not that familiar with Helm (mainly because I don't like it 😉). Are you able to provide a concrete suggestion, @eddycharly? 🙏

Reply:
@erikgb i pushed changes (fixed a bug at the same time)

Reply:
Thanks @eddycharly! 👍